source/documentation/deploying_services/use_a_custom_domain.erb

```diff
@@ -450,6 +450,10 @@ The above example configures the CDN route to have a default time-to-live of 15
 
 The CDN route service forwards cookie headers your app by default, so cookie-based authentication will work as expected. Other headers, such as HTTP auth, are stripped by default. If you need a different configuration, see the guidance on [forwarding headers](/deploying_services/use_a_custom_domain/#forwarding-headers).
 
+### Verifying protocols and cyphers
+
+You can find the TLS version and cyphers used on GOV.UK PaaS by using an SSL testing tool such as [SSL Labs](https://www.ssllabs.com/ssltest/). For your apps, use your CloudFront custom domain as the hostname.
```

Comment:
s/use your CloudFront custom domain/use your custom domain/

Reply:
ah yes, I can see that